public-pool-web/umbrel-app.yml

```
@@ -0,0 +1,25 @@
+manifestVersion: 1
+id: public-pool-web
+category: finance
+name: Public Pool's Web
+version: "0.0.7"
+tagline: Modern web interface for managing your Public Pool's data
+description: >
+  Public Pool's Web provides a sleek, user-friendly interface for managing and monitoring
+  your Public Pool's data. Built with Rails 8.0 and modern web technologies, it offers
+  real-time updates, detailed analytics, and comprehensive pool management features.
+  
+  This is a hobby project!
```

Adapt description formatting. @martinbarilik feel free to add some more points about features and so on.
```suggestion
  Public Pool's Web provides a sleek, user-friendly interface for managing and monitoring your Public Pool's data.
  
  Built with Rails 8.0 and modern web technologies, it offers real-time updates, detailed analytics, and comprehensive pool management features.


  This is a hobby project!
```